tests/referencesES6/advancedWithImports.jsx

```diff
@@ -1,4 +1,4 @@
-import { render, version, createVNode, createComponentVNode } from "inferno";
+import { cloneVNode, render, version, createVNode, createComponentVNode } from "inferno";
```

Comment:
seems these tests were also incorrectly compiled

Reply:
`ComponentType` is not been used as a value and `cloneVNode` is unused import. But maybe it is not a responsibility of this transformer to remove those?

Reply:
yeah this transformer shouldn't remove anything.